theme/themes/pxt/globals/site.variables

```diff
@@ -389,3 +389,7 @@
 @assetUnselected: #e3ddd0;
 @assetTypeButton: @teal;
 
+/*---------------------------
+   Tutorial Code Validation
+----------------------------*/
+@tutorialCodeValidationMoveOn: #B0F45A;
```

Comment:
set this to @green instead of a hardcoded CSS value

Reply:
If I were to make my own color theme for Tutorial Code Validation, can I hardcode CSS values?